parsers/ENTSOE.py

```diff
@@ -69,8 +69,28 @@
     },
     "storage": {"hydro storage": ["B10"]},
 }
+# ENTSOE production type codes mapped to their Electricity Maps production type.
 ENTSOE_PARAMETER_BY_GROUP = {
```

Comment:
Doesn't that make `ENTSOE_PARAMETER_GROUPS` irrelevant then? If yes then we should also remove it